src/Genres.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import './Genres.css';
+
+class GenreSelect extends Component {
```

Comment:
SearchForm has a couple of problems, like the when the input has focus then only enter key suppose to work and the current input value should be pass in callback is not followed.

You can just simply use generic form element of html, which will give you all the necessary  accessibility, you dont have to handle Enter key manually.